mineral-ui/mineral-ui
# Update Happo config to disambiguate components/subcomponents

### Description

* Updates Happo config to disambiguate components/subcomponents
* Pascal cases demo folder names

### Motivation and context

FIxes #536 

The folder renaming was needed in my original attempt to update the Happo config, wherein it was requiring example files directly.  I since changed tack and use a much simpler approach, using the demos/index.  The folder renames are not strictly needed here, but are probably better for consistency anyway.

### How to test

1. See the Happo report that should be generated for this PR.
2. Note that components and their subcomponents are listed separately, e.g. Card, Checkbox, Flex, Menu, Radio
3. Note that all components are included in report
4. Note that examples that have no source are not included, e.g. "Import Syntax"
5. I don't think that you can see in this report that examples with similar names, e.g. 'basic' or 'controlled' will be fixed, but we should see that after this is merged, or by running a report against this branch as the base.  Also, you should be able to see that images on the right hand side in the diffs show what we'd expect.

### Types of changes

- Other (provide details below)

Update to our test infrastructure

### Checklist
